symphony/app/fbcnms-projects/workflows/Dockerfile

```diff
@@ -29,4 +29,4 @@ WORKDIR /app/fbcnms-projects/workflows
 COPY fbcnms-projects/workflows .
 
 # Start app
-CMD yarn start
+CMD yarn run start:dev
```

Comment:
Let's leave the Dockerfile as is and create another dockerfile called `Dockerfile.dev` where this is the command. Then in `docker-compose.workflows.override.yml` you can override the dockerfile to be `Dockerfile.dev`.

This way the default is a production build and the override way is dev.

Reply:
Right. I'll just add it as `command` to `docker-compose.workflows.override.yml`